django_comments/compat.py

```diff
@@ -0,0 +1,6 @@
+import django
+
+if django.VERSION < (1, 7):
+    from django.contrib.sites.models import get_current_site
+else:
+    from django.contrib.sites.shortcuts import get_current_site
```

Comment:
@collinanderson  yes, totally agree.. but in version 1.7 and 1.8 I'm getting `RemovedInDjango19Warning: Please import get_current_site from django.contrib.sites.shortcuts.` in the console all the time

the intention of this pull request is to avoid these warnings :), but I can live with that if you guys prefer to use the `ImportError` pattern...

I thought this practice was widely accepted, I have seen this been used in many packages including DRF https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/compat.py#L143.

let me know your thoughts.